### ISSUE TITLE
[Help] Not receiving any results or errors from !ytsearch

### ISSUE THREAD

**Issue by jundis** [DuckBoss/JJMumbleBot#298]
<h2>Please have a look at the <a href="https://github.com/DuckBoss/JJMumbleBot/wiki/F.A.Q.">F.A.Q.</a> for solutions to common problems before posting an issue!</h2>

**Describe the bug**
Not receiving any results or errors, nothing in the logs, when using !ytsearch

**To Reproduce**
1. Install dependencies
2. CInstall v4.3.1
3. Configure and launch bot
4. Attempt to use !ytsearch and get nothing

**Expected behavior**
Results appear

**Screenshots**
![image](https://user-images.githubusercontent.com/766768/101442056-22a4fd80-38e0-11eb-9c52-5747a8e8b656.png)
![image](https://user-images.githubusercontent.com/766768/101442095-351f3700-38e0-11eb-8ddd-0e424f3413ac.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 16.04 LTS
 - Version 4.3.1
 - Python Version: 3.7.9

**Additional context**
These screenshots were while the script were running in verbose mode, yet nothing is logged.

**Comment by DuckBoss** [comment]
That's odd, based on your screenshot it looks like it's not parsing the command parameters correctly.
Thanks for bringing this to my attention, I'll look into it soon!

**Comment by MacLemon** [comment]
Can confirm this on my setup. Same behaviour.

Version information:
```
JJMumbleBot 4.3.1

$ python3.7 -V
Python 3.7.9

$ vlc --version
VLC media player 3.0.11 Vetinari (revision 3.0.11-0-gdc0c5ced72)

$ ffmpeg --version
ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
  built with FreeBSD clang version 8.0.1 (tags/RELEASE_801/final 366581) (based on LLVM 8.0.1)

$ youtube-dl --version
2020.11.26
```

Absolutely *nothing* in the logs other than the command received, also not when launching the bot with `-verbose`.

Soundboard seems to work fine. \o/

**Comment by DuckBoss** [comment]
> Absolutely nothing in the logs other than the command received, also not when launching the bot with -verbose.

Yes, currently logging is mostly used for core functions of the bot and hasn't been extensively used in plugins to debug 
commands. This is something I'll look into and work on so that <code>-verbose</code> logging has more details with plugin processing.


> Changing the youtube-dl dependency as listed in the version info doesn't change anything.

This isn't a problem with <code>youtube-dl</code> this time. It's an issue directly related to how youtube generates the web 
page for searches, and I am working on updating the web parsing to fix this.

**Comment by DuckBoss** [comment]
I've fixed the !ytsearch command issue in the newest PR, which will be merged and released in v4.3.2 soon.
